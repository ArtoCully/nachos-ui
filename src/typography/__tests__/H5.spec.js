import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import H5 from '../H5'

it('should render correctly', () => {
  const component = renderer.create(<H5>Text</H5>)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
