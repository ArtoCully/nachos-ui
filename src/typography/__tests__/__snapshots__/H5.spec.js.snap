exports[`test should render correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#bdc1cc",
        "fontSize": 15,
        "fontStyle": "normal",
        "fontWeight": "500",
        "paddingVertical": 10,
        "textAlign": "left",
      },
      undefined,
    ]
  }>
  Text
</Text>
`;
